sdk/mixedreality/arm-mixedreality/package.json

```diff
@@ -53,6 +53,9 @@
     "minify": "uglifyjs -c -m --comments --source-map \"content='./dist/arm-mixedreality.js.map'\" -o ./dist/arm-mixedreality.min.js ./dist/arm-mixedreality.js",
     "prepack": "npm install && npm run build"
   },
+  "//sampleConfiguration": {
+    "skipFolder": true
```

Comment:
@benbp This is the wrong package. Please make the change in https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/mixedreality/mixedreality-authentication/package.json instead

Reply:
Looks like it was already skipped in the other package, just missed the rename from `smokeTestConfiguration` to `sampleConfiguration` last month! Fixed now.

Reply:
Ah looks like a concurrent PR mishap (the package.json was created and committed around the same timeframe as the mass rename).